pkg/urbit/worker/main.c

```diff
@@ -422,6 +439,7 @@ _worker_lame(c3_d evt_d, u3_noun now, u3_noun ovo, u3_noun why, u3_noun tan)
     u3_noun lef = u3nc(c3__leaf, u3kb_weld(u3i_tape("bail: "),
                                            u3qc_rip(3, why)));
     u3_noun nat = u3kb_weld(u3k(tan), u3nc(lef, u3_nul));
+    _worker_send_tang(1, u3k(nat));
```

Comment:
This and the above changes will result in un-hushable error output on every event replacement.

Reply:
These can just be removed, right @belisarius222 ?